sesman/chansrv/clipboard.c

```diff
@@ -2494,7 +2521,14 @@ clipboard_xevent(void *xevent)
     switch (lxevent->type)
     {
         case SelectionNotify:
-            clipboard_event_selection_notify(lxevent);
+            if (g_outbound_clipboard_restricted == 0)
+            {
+                clipboard_event_selection_notify(lxevent);
+            }
+            else
+            {
+                return 1;
```

Comment:
I want some log here. Something like "Outbound clipboard is restricted by config.".

Reply:
Is it ok now?

Reply:
Is this comment enough? And is the logging level ok?

Reply:
Good enoug, thanks.